.github/workflows/codeql.yml

```
@@ -0,0 +1,69 @@
+on:
+  pull_request:
+    branches:
+      - github-actions
+
+jobs:
+  build_css:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source Git branch
+        uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)